joss-paper/paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'OpenTera WebRTC: A User Friendly WebRTC Wrapper for JavaScript, C++ and Python'
+tags:
+  - WebRTC
+  - C++
+  - Python
+  - JavaScript
+authors:
+  - name: Marc-Antoine Maheux
+    orcid: 0000-0002-3983-8754
+    affiliation: 1
+  - name: Dominic Létourneau
+    orcid: 0000-0001-7825-7533
+    affiliation: 1
+  - name: Philippe Warren
+    orcid: 0009-0008-4466-0963
+    affiliation: 1
+  - name: François Michaud
+    orcid: 0000-0002-3639-7770
+    affiliation: 1
+affiliations:
+  - name: Interdisciplinary Institute for Technological Innovation (3IT), Université de Sherbrooke, Canada
+    index: 1
+date: 15 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+During the COVID-19 pandemic, people were not able to socialize in person due to lockdowns, curfews and social distancing measures.
+So, social activities migrate to online platforms [@saltzman_loneliness_2020], such as videoconference ones.
+For example, people were remotely playing games with friends and family, but current technologies are not adapted to all types of games, such as tabletop games requiring non-verbal cues or containing a lot of physical components [@ramirez_gomez_socially_2022; @yuan_tabletop_2021].
+Telepresence robots are another type of technology providing a way to stay in touch with loved ones.
+Compared to videoconference platforms, the remote user is usually able to move in a remote environment and talk with a remote person [@kristoffersson_review_2013].
+They are used in aged care settings to address loneliness [@hung_facilitators_2022], but ???.
+Thus, technologies providing a way to stay in touch with people require future research to improve the platforms in contexts like remote gaming and aged care setting.
+Doing so needs to develop new platforms because commercially available platforms are closed to modifications [???].
+
+WebRTC is the de facto technology to develop videoconference platforms since it is secured and built for web browsers, hence accessible.
+Video streams, audio streams and data can be exchanged between the participants of a call.
+A JavaScript API is available in modern web browsers, but the developer needs to create a web server to initialize the peer-to-peer calls, which is usually named a signaling server.
+The library used by most web browsers to include WebRTC support is the WebRTC Native library.
+In the context of remote tabletop gaming, adding sensors to the videoconference setups to improve the gaming experience is limited by their compatibility with web browsers.
+On the other hand, a web interface built with WebRTC can control a telepresence robot, but the robot needs to run a web browser for the JavaScript API, which is not great in resource-constrained robots.
```

Comment:
the robot needs to run the equivalent of a web browser for the Javascript API. It is often done via the integration of Chromium web engine (Electron, Qt) and requires code in Javascript, which might not be optimized.